search/src/main/res/layout/fragment_search_audio_detail.xml

```diff
@@ -0,0 +1,165 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/audioBackground"
+    android:fitsSystemWindows="true"
+    app:statusBarBackground="@android:color/transparent">
+
+    <android.support.design.widget.AppBarLayout
```

Comment:
If this is the same as one of the other layouts' AppBarLayouts, then we should probably pull that out into an `include`.

Reply:
Yes this can be done in a different merge. Will create a chore ticket for it.